src/opera/pge/dswx_s1/dswx_s1_pge.py

```diff
@@ -27,6 +32,46 @@ class DSWxS1PreProcessorMixin(PreProcessorMixin):
 
     _pre_mixin_name = "DSWxS1PreProcessorMixin"
 
+    def _validate_algorithm_parameters_config(self):
+        """
+        The DSWx-S1 interface SAS uses two runconfig files; one for the main SAS,
+        and another for algorithm parameters.  This allows for independent modification
+        of algorithm parameters within it's own runconfig file. This method performs validation
+        of the 'algorithm parameters' runconfig file against its associated schema file. The SAS
+        section of the main runconfig defines the location within the container of the 'algorithm
+        parameters' runconfig file, under ['dynamic_ancillary_file_group']['algorithm_parameters'].
+        The schema file for the 'algorithm parameters' runconfig file is referenced under
+        ['PrimaryExecutable']['ParametersSchemaPath'] in the PGE section of the runconfig file.
+
+        """
+        self.runconfig = RunConfig(self.runconfig_path)
```

Comment:
Since `DSWxS1PreProcessorMixin.run_preprocessor` runs this function after the pre-processor activity of the base PGE, I'm not sure this line is necessary. This method should be able to assume the "main" RunConfig has already been parsed, validated and assigned to `self.runconfig`

Reply:
removed